tests/Request/MultilingualTraitTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * @file
+ * Contains \Triquanta\IziTravel\Tests\Request\MultilingualTraitTest.
+ */
+
+namespace Triquanta\IziTravel\Tests\Request;
+
+use Triquanta\IziTravel\Request\MultilingualTrait;
+
+/**
+ * @coversDefaultClass \Triquanta\IziTravel\Request\MultilingualTrait
+ */
+class MultilingualTraitTest extends \PHPUnit_Framework_TestCase {
+
+    /**
+     * The language codes.
+     *
+     * @var string[]
+     *   An array of ISO 639-1 alpha-2 language codes.
+     */
+    protected $languageCodes = [];
+
+    /**
+     * An invalid language codes.
+     *
+     * @var string
+     */
+    protected $invalidLanguageCodes;
```

Comment:
The property name and its docblock description are plural, yet the type hint indicates a single string value.